tests/utils/testHelpers.ts

```diff
@@ -0,0 +1,85 @@
+import { expect } from "chai";
+import { hex_to_bytes } from "./convertions";
+import Contract from "../../typed_contracts/multisig/contracts/multisig";
+import Constructors from "../../typed_contracts/multisig/constructors/multisig";
+import { Transaction } from "../../typed_contracts/multisig/types-arguments/multisig";
+
+let init_threshold = 2;
+
+export const assignKeyringPairs = (keyring) => {
```

Comment:
Maybe generate an array of all the possible accounts in the helper as a const, and then in the function you receive an int as a parameter and return a slice of that array.